Ultrasonic/Ultrasonic.cpp

```diff
@@ -13,43 +13,32 @@
 #include "Ultrasonic.h"
 
 Ultrasonic::Ultrasonic(int TP, int EP)
-{
-   pinMode(TP,OUTPUT);
-   pinMode(EP,INPUT);
-   Trig_pin=TP;
-   Echo_pin=EP;
-   Time_out=3000;  // 3000 µs = 50cm // 30000 µs = 5 m 
-}
+  :trigPin(TP), echoPin(EP), timeOut(3000){}
 
 Ultrasonic::Ultrasonic(int TP, int EP, long TO)
-{
-   pinMode(TP,OUTPUT);
-   pinMode(EP,INPUT);
-   Trig_pin=TP;
-   Echo_pin=EP;
-   Time_out=TO;
+  :trigPin(TP), echoPin(EP), timeOut(TO){}
+
+void Ultrasonic::Config() {
+  pinMode(this->trigPin, OUTPUT);
+  pinMode(this->echoPin, INPUT);
 }
 
-long Ultrasonic::Timing()
-{
-  digitalWrite(Trig_pin, LOW);
+void Ultrasonic::Timing() {
+  digitalWrite(this->trigPin, LOW);
   delayMicroseconds(2);
-  digitalWrite(Trig_pin, HIGH);
+  digitalWrite(this->trigPin, HIGH);
   delayMicroseconds(10);
-  digitalWrite(Trig_pin, LOW);
-  duration = pulseIn(Echo_pin,HIGH,Time_out);
-  if ( duration == 0 ) {
-	duration = Time_out; }
-  return duration;
+  digitalWrite(this->trigPin, LOW);
+
+  this->duration = pulseIn(this->echoPin, HIGH, this->timeOut);
+
+  if (duration == 0)
+	 this->duration = this->timeOut;
+   return;
 }
 
-long Ultrasonic::Ranging(int sys)
-{
+long Ultrasonic::Ranging(int sys) {
   Timing();
-  if (sys) {
-	distance_cm = duration /29 / 2 ;
-	return distance_cm;
-  } else {
-	distance_inc = duration / 74 / 2;
-	return distance_inc; }
+	return sys ? this->duration /29 / 2 :

```

Comment:
Reduces readability, but it is ok for me. I would get rid of `/29 /2` because that's strange as he** and I would replace it for `/29*2`. Same applies for `/74 /2`.